enterprise/server/remote_execution/containers/docker/docker.go

```diff
@@ -166,27 +173,30 @@ func (r *dockerCommandContainer) Run(ctx context.Context, command *repb.Command,
 			return result
 		}
 		result.ExitCode = int(s.StatusCode)
-		logOptions := dockertypes.ContainerLogsOptions{
-			ShowStdout: true,
-			ShowStderr: true,
-		}
-		logs, err := r.client.ContainerLogs(ctx, cid, logOptions)
-		if err != nil {
-			result.Error = wrapDockerErr(err, "failed to get docker container logs")
-			return result
-		}
-		err = copyOutputs(logs, result)
-		if closeErr := logs.Close(); closeErr != nil {
-			log.Warningf("Failed to close docker logs: %s", closeErr)
-		}
-		if err != nil {
-			result.Error = wrapDockerErr(err, "failed to read docker container logs")
-			return result
-		}
 	}
 	return result
 }
 
+func (r *dockerCommandContainer) containerLogs(ctx context.Context, cid string, result *interfaces.CommandResult) {
```

Comment:
nit: would maybe be nice to call this something like copyContainerLogs or addContainerLogs to make it clear they are being read and added to the result?

Reply:
good point, I mindlessly copied the docker API name. Done